AuxTel/Standard-Operations/Daytime-Operations/latiss-daily-calibrations-BIAS-DARK-FLAT-all-filters-empty-procedure.rst

```diff
@@ -43,232 +45,246 @@ Procedure Steps
 #. Enable ``Scheduler:2`` with a valid scheduler configuration. Use the standard script :file:`auxtel/scheduler/enable.py` with the configuration required for the run and available in the corresponding night log. 
 #. Run the script :file:`add_block.py` to the ATQueue  with the following configuration:
 
-.. code-block:: text
-  :caption: :file:`add_block.py`
+    .. code-block:: text
+      :caption: :file:`add_block.py`
+
+      id: setup_latiss_calibrations
+
+    The BLOCK with :file:`setup_latiss_calibrations` configuration will queue the scripts aimed to setup the system. It will start with the :file:`set_summary_state.py` script to enable ATWhiteLight CSC with the following configuration:
 
-  id: latiss_daily_calibrations
+    .. code-block:: text
+      :caption: :file:`set_summary_state.py`
+    
+      data:
+        -
+          - ATWhiteLight 
+          - ENABLED 
 
+    The :file:`set_summary_state.py` script will enable ``OCPS:1`` CSC.
 
-The BLOCK with :file:`latiss_daily_calibrations` configuration will queue the following scripts:
+    .. code-block:: text
+      :caption: :file:`set_summary_state.py`
+    
+      data:
+        -
+          - OCPS:1
+          - ENABLED
 
-The BLOCK will start with the :file:`set_summary_state.py` script to enable ATWhiteLight CSC with the following configuration:
+    The SAL script :file:`auxtel/calibrations/power_on_atcalsys.py` with no configuration inserted into the BLOCK :file:`setup_latiss_calibrations` structure will start the chiller, turn on the white light and open the shutter.  This script takes 15 minutes to complete, the time it takes to warm up the white light. On Chronograf using the query :file:`lsst.sal.ATWhiteLight.logevent_logMessage.message`, the lamp reports its retry loop status explicitly.
 
-.. code-block:: text
-  :caption: :file:`set_summary_state.py`
- 
-  data:
-     -
-       - ATWhiteLight 
-       - ENABLED 
+    .. code-block:: text
+      :caption: :file:`auxtel/calibrations/power_on_atcalsys.py`
 
-The :file:`set_summary_state.py` script will enable ``OCPS:1`` CSC.
+    The :file:`auxtel/prepare_for/flat.py` script (empty configuration) will position the telescope and dome in FLAT position. The telescope will point towards the dome flat screen (mount Az = 188.7 deg, mount El = 39.0 deg, dome Az= 2.59 deg). Confirm in the cameras aux-cam01 or aux-cam02 that the white light is on, and telescope is pointing to the dome flat screen.
```

Comment:
Note: We need to document and link here how to access the cameras.

Reply:
Note added to the .rst file.